### ISSUE TITLE
For GNN, Use DataLoader to split the data into train and test

### ISSUE THREAD

**Issue by ShikhaAtGitHub** [KanishkGoel1999/Group-8-Capstone-#11]
Utilize PyTorch Geometric's DataLoader to efficiently split graph data into training and test sets for Graph Neural Networks (GNNs). Ensure proper batching and shuffling mechanisms for optimal model training and evaluation.
Use pytorch geometric train test split

**Comment by ShikhaAtGitHub** [comment]
We have added the below changes:

RandomNodeSplit(split="train_rest", num_val=0.1, num_test=0.1)

**Comment by ShikhaAtGitHub** [comment]
Added NeighborLoader.
